reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+
+.. warning::
+    As these APIs are mainly designed to be used by connectors, Enterprise Edition permissions are not managed.
+
+Security
+--------
+
+We use `OAuth2 <https://oauth.net/2/>`_ to authenticate users in the API and
+`Symfony ACLs <http://symfony.com/doc/2.7/security/acl.html>`_ to handle authorizations.
+
+Authentication
+~~~~~~~~~~~~~~
+
+Creating an OAuth client
+************************
+
+The first operation you will need to authorize third-party applications to use the API is to create a `client id`.
+To do so, use the following command:
+
+.. code-block:: bash
+
+    php bin/console pim:oauth-server:create-client \
+        --grant-type="password" \
+        --grant-type="refresh_token"
+
+You will receive the client id and client secret:
+
+.. code-block:: bash
+
+    A new client with client id 4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s and secret 5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s has been added.
+
+These keys must be transmitted by the administrator to any third-party application wanting to use the API.
+
+Client ids are not enough to access the API, it's the role of tokens. Client ids allow two things:
+asking a token for a specific user, and refresh an expired token.
+
+.. note::
+  In the case you want to give a time-limited access, you can create the client id without specifying the
+  `--grant-type="refresh_token"` option. This way, the client application can only access the API until the first token
+  expires, then will be unable to refresh it.
+
+Revoking a client
+*****************
+
+To revoke a client id use the following command:
+
+.. code-block:: bash
+
+    php bin/console pim:oauth-server:revoke-client 4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s
+
+.. warning::
+  Client ids can be revoked at any moment by the administrator. In this case, all tokens created with the
+  revoked client id will be invalidated and the application using this client id will be unable to ask a new one.
+
+Listing all client ids
+**********************
+
+To get the list of all existing client ids use the following command:
+
+.. code-block:: bash
+
+    php bin/console pim:oauth-server:list-clients
+
+.. code-block:: bash
+
+    +----------------------------------------------------+----------------------------------------------------+
+    | Client id                                          | Secret                                             |
+    +====================================================+====================================================+
+    | 3e2iqilq2ygwk0ccgogkcwco8oosckkkk4gkoc0k4s8s044wss | 44ectenmudus8g88w4wkws84044ckw0k4w4kg0sokoss84oko8 |
+    +----------------------------------------------------+----------------------------------------------------+
+
+Getting a token
+***************
+
+A client id has been created and set into the client application. The last information needed to use the API is the token.
```

Comment:
"and provided to the client application" instead "of set into the client application"